### ISSUE TITLE
Deprecated elements: Details of illustrative content and Details of color content

### ISSUE THREAD

**Issue by jess-grz** [rbms-bsc/DCRMR#150]
The elements [Details of illustrative content](https://bsc.rbms.info/DCRMR/phys-desc/Details-of-illustrative-content/) and [Details of color content](https://bsc.rbms.info/DCRMR/phys-desc/Details-of-color-content/) became deprecated in RDA shortly before the initial release of DCRMR. We decided to proceed with including them, adding a note to the element pages that the elements were deprecated but that we retained them "to mirror the structure throughout the Physical description chapter." We indicated that Details of illustrative content should be mapped to RDA element Illustrative content and that Details of color content should be mapped to RDA element Color content.

We should avoid the use of deprecated elements whenever possible. Inevitably, RDA elements will become deprecated before we can address them fully in DCRMR, but retaining them should probably be a temporary stopgap rather than a permanent solution. I don't think there is a rare materials-specific reason for retaining these two particular elements, and we'll need to consider how to incorporate their instructions into other elements. Bu we likely cannot take this work on until after the graphics release.

The simplest solution seems to be to incorporate the instructions into [Illustrative content](https://bsc.rbms.info/DCRMR/phys-desc/Illustrative-content/) and [Color content](https://bsc.rbms.info/DCRMR/phys-desc/Color-content/). RDA does allow for unstructured descriptions in both of these elements (the sorts of notes that we're accustomed to making for Details of illustrative content and Details of color content).

There is an extensive closed issue related to color content (#33) that may be helpful to consult. Also see Gordon Dunsire's paper from December 2019, _[RDA content elements](http://www.rda-rsc.org/sites/all/files/RDA%20Content%20elements%20followup.pdf)_, for more info.

**Comment by adams-liz** [comment]
In Jessica and my initial email thread about this, I had come down on the side that we should move Details of illustrative content and Details of color content to Notes on manifestation (similar to parts of the discussion in #33). I think after reading through that closed issue and looking again at RDA, that it makes sense to move these to Illustrative content and Color content.

**Comment by ladylazarus3** [comment]
i agree with @adams-liz that the content for these depreciated RDA elements should be moved to Illustrative content and Color content.  

kalan

**Comment by lizoknj** [comment]
Agree that the content for these two elements should be moved to Illustrative content and Color content. 
Wondering how they would be integrated within these elements. Would they be treated in a section at the end of the elements, under the heading of "unstructured" or "free-text" information? Not sure.

**Comment by jess-grz** [comment]
@lizoknj, I imagine that we'll need to assemble a small subgroup when we're ready to tackle this. It seems simple at first glance to incorporate the instructions for the two deprecated elements, but the process of folding them into the other elements in a way that makes logical, grammatical, and stylistic sense will be complicated. It's nothing that this group can't handle, of course :)

**Comment by io-uk** [comment]
I agree that the content for these two elements should be incorporated into Illustrative content and Colour content somehow. 

Just mentioning this here to have a record of it: I have looked at the elements Colour content and Illustrative content in the RDA Toolkit for my assignment to propose policy statements for phys. description elements. If we decide to include links back to DCRMR or refer to specific DCRMR instruction numbers when we first publish our policy statements, we will then have to adapt these links/references once we have moved/incorporated the details sections into their respective elements.

And on another note: Currently, the element reference for “color content” in the RDA Toolkit doesn't contain the 500 MARC field in the MARC 21 Bibliographic mapping at all. I have flagged this with Thurstan Young and he agreed that the 500 should be added to the MARC mapping in colour content to cover an "unstructured description". This field has now been added to the MARC mapping but it will only show up in the RDA Toolkit after the next RDA Toolkit release.
